prow/e2e-suite.sh

```diff
@@ -91,4 +94,4 @@ E2E_ARGS+=("--skip_cleanup")
 time ISTIO_DOCKER_HUB=$HUB \
```

Comment:
Apparently I can't comment on lines you didn't edit, so commenting here...  I think you need to set the TAG on line 81 to use the variant, then you don't need the changes to `istio_presubmit.sh` or need to call it in https://github.com/istio/test-infra/pull/1352.

Sorry you got caught in our transition in prow to deprecate the  `istio_presubmit.sh` so things are a bit more complicated right now

Reply:
We changed line 80/81 accordingly